6.5.set.py

```diff
@@ -0,0 +1,43 @@
+
+import random
+from itertools import combinations
+
+def create_cards():
+    cards = []
+
+    for shape in range(1, 4):
+        for shade in range(1, 4):
+            for color in range(1, 4):
+                for count in range(1, 4):
+                    cards.append({'shape': shape, 'shade': shade, 'color': color, 'count': count})
```

Comment:
This part doesn't look good you have few solutions:
1. Use a comprehension instead -
https://www.geeksforgeeks.org/comprehensions-in-python/

2.Replace these two for loops with a list comprehention with two iterators - 
https://stackoverflow.com/questions/18551458/how-to-frame-two-for-loops-in-list-comprehension-python